servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -30,7 +41,19 @@
  */
 public final class OutlierDetectorConfig {
```

Comment:
This is part of the most important change in this PR, specifically it is now the core API used to configure all types of outlier detection.